services/121-service/src/seed-data/fsp/fsp-intersolve-visa.json

```diff
@@ -7,7 +7,7 @@
   "notifyOnTransaction": true,
   "questions": [
     {
-      "name": "addressStreet",
+      "name": "street",
```

Comment:
I found that several files still contain the old questions names. For example:
  - test/registrations/pagination/pagination-data.ts
  - src/registration/enum/custom-data-attributes.ts 
  - there is no migrations of the fsp questions

Not sure if changing these files should also be parts of this commit

I am also wondering if these changes are these changes really worth it? All fields that are send to the 'address' key of Intersolve are prefixed with address seems pretty consistent and does not impact the readability of our code. 

Releasing this requires quite a bit of coordination:
- Kobo/Espocrm connect needs to be updated immediately after release or schools that register additional parents will not go through
- Sanne needs to do a data migration in the 121-platform
  - I am not even sure if you can still do an export of not included fsp question any more

Reply:
@RubenGeo  wins. This time. :)